layouts/shortcodes/competition-livestream.html

```diff
@@ -18,5 +18,3 @@ <h1>Competition Live!</h1>
     <p>The competition is underway at the University of Southampton.</p>
     <p>See the <a href="/comp/">latest scores</a>.</p>
 </div>
-
-<h2>About Us</h2>
```

Comment:
Why are we removing this header?

It's here to provide a gap between the stream and the following paragraph; it was deliberately in this file as we won't need it when we stop using this file.

Reply:
aa90afee66a3a18ff88494dfc234f04d52bb67d7 undoes this.

Reply:
It only makes sense when content below it is an about us section. Remembering to add a `## About Us` under it for the index page feels like a much nicer fix?

Reply:
It seems improbable that we'd deploy this without checking it. The reality is that we do currently have an about section as the top item on the homepage.

This is all rather moot if we're going to remove this partial anyway.